packages/video_player/video_player/README.md

```diff
@@ -135,4 +135,29 @@ and so on.
 
 To learn about playback speed limitations, see the [`setPlaybackSpeed` method documentation](https://pub.dev/documentation/video_player/latest/video_player/VideoPlayerController/setPlaybackSpeed.html).
 
+### Picture-in-Picture
+
+#### iOS
+On iOS the picture-in-picture is linked to the AVPlayerController.
+If you want to enable picture-in-picture make sure to enable the `audio` capability (in Xcode's UI it will say **Audio, AirPlay, and Picture in Picture**).
+Not setting this capability but calling `setPictureInPictureOverlayRectMessage` and `setPictureInPicture` will not start the picture-in-picture.
+
+```xml
+    <key>UIBackgroundModes</key>
+    <array>
+        <string>audio</string>
+    </array>
+```
+
+Example:
+![The example app running in iOS with picture-in-picture enabled](https://github.com/flutter/plugins/blob/main/packages/video_player/video_player/doc/demo_pip_iphone.gif?raw=true)
+
+#### Android
+
+On Android there is no link to the video player. Your complete app will be minimized ([picture-in-picture Android documentation](https://developer.android.com/guide/topics/ui/picture-in-picture))
```

Comment:
Picture in picture is not tightly coupled to the video player, as it is the case for iOS

Reply:
@stuartmorgan anything I should do about this? Or do you have any more feedback for this comment?

Reply:
I would replace this whole section with something like "On Android, picture-in-picture mode is implemented at the application level rather than the video element level, so this plugin does not implement picture-in-picture mode on Android."